### PR TITLE
Uploading myHacktoberfestFlower art files

### PR DESCRIPTION
**Pull request by betiniakarandut**
# Zero To Mastery - Canvas 2022 🎉🎉

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
Submitting my canvas art files for myHacktoberfestFlower art. Thank you

## Breaking changes
None

## Screenshots
icon

### CODE REVIEW

File: src/art/betiniakarandut/meta.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
+<<<<<<< HEAD
```
Comment (sophiabrandt) [inline src/art/betiniakarandut/meta.json:2]: Please fix this file. That's not valid json.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
You are overriding a previous contribution. Was that intentional?

If not, make a new folder with a variation of your name, please.

File: src/art/betiniakarandut/meta.json (hunk 1 above)
Comment (betiniakarandut) [reply src/art/betiniakarandut/meta.json:2]: Thank you for the quick observation. I have changed the folder name from betiniakarandut to betiniakarandut01.
Had merge conflict issues before and forgot to resolve it in the meta.json foldrer too. PENDING ISSUES RESOLVED.

**Review by auxfuse — APPROVED** [review]
LGTM 👾 🎇

**Pull request merged**